Programmers/230914_귤_고르기/kwaksj329.js

```diff
@@ -0,0 +1,20 @@
+function solution(k, tangerine) {
+  let answer = 0;
+  let tangerineSum = tangerine.reduce((acc, curr) => {
+    curr in acc ? (acc[curr] += 1) : (acc[curr] = 1);
+    return acc;
+  }, {});
+
+  let valueSorted = Object.keys(tangerineSum)
+    .map((item) => tangerineSum[item])
+    .sort((a, b) => b - a);
```

Comment:
```suggestion
const valueSorted = Object.values(tangerineSum).sort((a, b) => b - a);
```
key는 필요 없고 value만 사용되고 있는 상황이라, 이런 경우에는 `Object.values`를 사용하면 더 간단해요!